agent/unit-test/configstore_unittest.cpp

```diff
@@ -118,6 +118,20 @@ TEST_F(ConfigStoreTest, readConfigFetchOneCap) {
     ASSERT_FALSE(retGWs.empty());
 }
 
+/* Getting all capability names after reading a service manifest should
+ * result in getting all those names that are listed in the manifest file.
+ */
+TEST_F(ConfigStoreTest, readConfigFetchAllCaps) {
+    FilteredConfigStore cs = FilteredConfigStore(testDataDir + sm_path);
```

Comment:
Please take the opportunity to change the variables that are named differently then the others (and also not according to the style guide).